### PR TITLE
Improve sample

### PR DESCRIPTION
**Pull request by AlexBalo**

### CODE REVIEW

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java [hunk 1]
```diff
@@ -349,6 +360,37 @@ public void onItemClick(Object item, View view, int position) {
...
     }
 
+    public void loopViewsRecursively(View parent) {
+        if (parent == null) {
+            return;
+        }
+
+        if (parent instanceof ViewGroup) {
+            ViewGroup viewGroup = (ViewGroup) parent;
+            for (int i = viewGroup.getChildCount() - 1; i >= 0; i--) {
```
Comment (orhanobut) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: I would use local variable for count.
<pre>int count= viewGroup.getChildCount() </pre>

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java [hunk 2]
```diff
@@ -434,6 +476,7 @@ public boolean onTouch(View v, MotionEvent event) {
         private Gravity gravity = Gravity.BOTTOM;
         private ScreenType screenType = ScreenType.HALF;
         private OnItemClickListener onItemClickListener;
+        private OnClickListener onClickListener;
         private int inAnimation = INVALID;
```
Comment (orhanobut) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: I would put an empty line in order to separate primitive and reference types.

**Comment by orhanobut** [comment]
With the real samples, it looks really great :) finished reviewing

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java (hunk 1 above)
Comment (AlexBalo) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: Yeah, sure. I forgot that.

File: dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java (hunk 2 above)
Comment (AlexBalo) [inline dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java]: Right, I will also move the others and separate with empty line

**Comment by AlexBalo** [comment]
Finished fixing issues and pushed again

**Comment by orhanobut** [comment]
LGTM

**Pull request merged**